ext/storage_sources/s3_store/aws_bucket_conn.h

```diff
@@ -18,7 +18,7 @@ class aws_bucket_conn {
     explicit aws_bucket_conn(const Aws::S3Crt::ClientConfiguration &config);
     bool list_buckets(std::vector<std::string> &buckets) const;
     bool list_objects(
-      const std::string bucket_name, std::vector<Aws::S3Crt::Model::Object> &bucket_objects) const;
+      const std::string &bucket_name, std::vector<Aws::S3Crt::Model::Object> &bucket_objects) const;
     bool put_object(const Aws::String &bucket_name, const Aws::String &object_key,
```

Comment:
Shouldn't it be `std::string` here instead of `Aws::String` for the different arguments? [The body](https://github.com/wiredtiger/wiredtiger/pull/7449/commits/f7a8589378078e4c174a5ac7a36b836d25039d27#diff-8072338eb0ed9f4f2ad4d11ba96582a3713c5970c02296b8425ad14f38eae230R63) is defined differently, did it compile without any warning?

Reply:
Yes it still compiled without any warning. Thanks for spotting this difference, changed now!